### PR TITLE
Backport upstream changes

### PR DESCRIPTION
**Pull request by alfonsosanchezbeato**
Upstream MPs are now merged, make sure we are in sync.

### CODE REVIEW

**Comment by mvo5** [comment]
Do you have a link to the upstream MPs maybe for completeness? 

[edit: Nevermind, sorry for the noise. I see it's part of the new patches]

**Review by stulluk — APPROVED** [review]
LGTM

File: plymouth-patch/0002-Add-ThemDir-configuration-option.patch [hunk 1]
```diff
@@ -1,43 +1,49 @@
...
+From: snapcraft <snapcraft@canonical.com>
+Date: Mon, 27 Sep 2021 15:20:30 +0200
+Subject: [PATCH 2/2] Add ThemDir configuration option
 
-Add a new option called ThemePath to the configuration file, so a
+Add a new option called ThemeDir to the configuration file, so a
 configurable path can be used to ship themes.
 
-Gbp-Pq: Name 0017-Add-ThemePath-configuration-option.patch
+Upstream MP: https://gitlab.freedesktop.org/plymouth/plymouth/-/merge_requests/142
```
Comment (mvo5) [inline plymouth-patch/0002-Add-ThemDir-configuration-option.patch:9]: Thanks for adding those.

**Review by mvo5 — APPROVED** [review]
Very nice!

**Pull request merged**